apps/src/gamelab/AnimationTab/AnimationTab.jsx

```diff
@@ -7,7 +7,7 @@ import AnimationPicker from '../AnimationPicker/AnimationPicker';
 import GameLabVisualizationHeader from '../GameLabVisualizationHeader';
 import { setColumnSizes } from './animationTabModule';
 import AnimationList from './AnimationList';
-import ResizablePanes from './ResizablePanes';
+import ResizablePanes from '../../templates/ResizablePanes';
```

Comment:
At two layers deep, maybe better now to refer to `@cdo/apps/templates/ResizablePanes`?